src/main/scala/uk/gov/hmrc/sbtcompilesass/SbtCompileSass.scala

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmrc.sbtcompilesass
+
+import sbt.*
+import sbt.Keys.*
+import com.typesafe.sbt.web.SbtWeb.autoImport.*
+import com.typesafe.sbt.web.Import.WebKeys.*
+import com.typesafe.sbt.web.*
+import de.larsgrefer.sass.embedded.SassCompilerFactory
+import scala.collection.JavaConverters._
+
+import java.nio.file.Files
+import scala.util.Using
+
+/*
+  This compiles sass in the same way as sbt-sassify but using dart-sass
+
+  Unlike SbtSassify this does not do incremental compilation, is that only about speed (maybe not needed)
+  or is it also about recompiling when changes are made in development?
+ */
+object SbtCompileSass extends AutoPlugin {
+  override def requires: Plugins      = SbtWeb
+  override def trigger: PluginTrigger = AllRequirements
+
+  object autoImport {
+    val compileSass = TaskKey[Seq[File]]("compileSass", "Create css files from scss and sass files.")
+  }
+
+  import autoImport._
+
+  override def projectSettings = Seq(
+    // exclude sass files from being copied into the target dir by SbtWeb
```

Comment:
this is actually happening with sbt-sassify at the moment

for example https://www.tax.service.gov.uk/tracking-consent/assets/stylesheets/application.scss